### PR TITLE
Displayservice

### PR DESCRIPTION
**Pull request by thedetukov**
Implement DisplayService, DisplayDevice.

https://github.com/zxteamorg/desktop-preferences/issues/9 - посилання на план завдання

### CODE REVIEW

File: lib/src/tools/display/display_service_contract.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  ///
+  List<DisplayDevice> get displays;
+}
+
+abstract class DisplayDevice {
+  /// Name of the Display device
+  String get name;
+
+  /// brightness of the Display device
+  int get brightness;
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:51]: What brightness typeof `int`?  I expect `double`.

File: lib/src/tools/display/display_service_contract.dart [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  ///
+  /// Call this method to set brightness for one display
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display adapter is turn ON.
+  bool get isDarkModeEnabled;
+
+  ///
+  /// Call this method to disable Display adapter
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:11]: Is it proper comment?

File: lib/src/tools/display/display_service_contract.dart [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  ///
+  void disableDarkMode() {
+    this.isDarkModeEnabled = false;
+  }
+
+  ///
+  /// Call this method to enable Display adapter
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:18]: Is it proper comment?

File: lib/src/tools/display/display_service_contract.dart [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  ///
+  void enableDarkMode() {
+    this.isDarkModeEnabled = true;
+  }
+
+  bool get isNightModeEnabled;
+
+  ///
+  /// Call this method to disable Display adapter
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:27]: Is it proper comment?

File: lib/src/tools/display/display_service_contract.dart [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  ///
+  void disableNightMode() {
+    this.isNightModeEnabled = false;
+  }
+
+  ///
+  /// Call this method to enable Display adapter
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:34]: Is it proper comment?

File: lib/src/tools/display/display_service_contract.dart [hunk 6]
```diff
@@ -0,0 +1,52 @@
+abstract class DisplayService {
+  ///
+  /// Call this method to set brightness for one display
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display adapter is turn ON.
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:7]: Is it proper comment?

File: lib/src/tools/display/display_service_contract.dart [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  ///
+  /// Call this method to enable Display adapter
+  ///
+  void enableDarkMode() {
+    this.isDarkModeEnabled = true;
+  }
+
+  bool get isNightModeEnabled;
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:24]: Where is a comment?

**Review by theanurin — CHANGES_REQUESTED** [review]

File: lib/src/tools/display/display_service_contract.dart [hunk 8]
```diff
@@ -0,0 +1,53 @@
+abstract class DisplayService {
+  ///
+  /// Call this method to set brightness for one display.
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display Dark Mode is turn ON.
```
Comment (theanurin) [inline lib/src/tools/display/display_service_contract.dart:7]: Please use three lines for comment (to follow style)

```
///
/// Gets whether Display Dark Mode is turn ON.
///
```

File: lib/src/tools/display/display_service_contract.dart (hunk 1 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed the variable type from int to double.

File: lib/src/tools/display/display_service_contract.dart (hunk 2 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed comment:  "Call this method to disable Display Dark Mode".

File: lib/src/tools/display/display_service_contract.dart (hunk 3 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed comment: "Call this method to enable Display Dark Mode.".

File: lib/src/tools/display/display_service_contract.dart (hunk 4 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed comment: "Call this method to disable Display Night Mode.".

File: lib/src/tools/display/display_service_contract.dart (hunk 5 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed comment: "Call this method to enable Display Night Mode.".

File: lib/src/tools/display/display_service_contract.dart (hunk 6 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart]: Changed comment: "Gets whether Display Dark Mode is turn ON.".

File: lib/src/tools/display/display_service_contract.dart (hunk 8 above)
Comment (thedetukov) [reply lib/src/tools/display/display_service_contract.dart:8]: Added  three lines for comment.

**Review by theanurin — APPROVED** [review]
Looks good

**Pull request merged**